docs/v0.7/en/greptimecloud/usage-&-billing/dedicated.md

```diff
@@ -0,0 +1,34 @@
+# Dedicated Plan
+
+The dedicated plan offers unlimited data storage and retention,
+completely isolated resource control and network,
+and includes support from Greptime's SRE team.
+
+If you require absolute isolation from other users or 
+need to exceed the maximum WCU and RCU limits of the Serverless Plan, 
+then the Dedicated Plan is your choice.
```

Comment:
Shall we use "Dedicated Plan" in this case form everywhere?

Reply:
I'm not sure what you mean..?

Reply:
In the starting paragraph we use "dedicated plan" instead of "Dedicated Plan".